patchkit-launcher-qt-src/src/config.cpp

```diff
@@ -56,7 +56,8 @@ const QString Config::pingCountArg = "-n";
 const QString Config::pingCountArg = "-c";
 #endif
 
-const int Config::lockingTimeout = 100;
+const int Config::staleLockTime = 30000;
```

Comment:
30 seconds is the default value for stale time in QT.

I've left the value here as a customization option.